sdk/python/feast/repo_config.py

```diff
@@ -414,7 +414,6 @@ def _validate_feature_server_config(cls, values: Any) -> Any:
         return values
 
     @field_validator("project")
-    @classmethod
```

Comment:
https://docs.pydantic.dev/2.6/api/functional_validators/#pydantic.functional_validators.model_validator
just checked here, actually we do need `classmethod`